packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -898,15 +881,16 @@ class RenderOpacity extends RenderProxyBox {
 
   @override
   void paint(PaintingContext context, Offset offset) {
-    if (_alpha == 0) {
-      return;
-    }
-    super.paint(context, offset);
+    layer = context.pushOpacity(needsCompositing, offset, _opacity, super.paint, oldLayer: layer as OpacityLayer?, size: size);
```

Comment:
nit: maybe add some line breaks to this to make it a little easier to read (add trailing comma and then one property per line)

Reply:
Done